src/pages/Home/index.tsx

```diff
@@ -20,15 +24,24 @@ export const HomeUserGraphQL = graphql`
 
 const Home = (props: PagePropsWithQuery<HomeUserQuery>) => {
     const data = usePreloadedQuery<HomeUserQuery>(HomeUserGraphQL, props.queryReference);
+
+    const { t } = useTranslation(["common", "home"]);
```

Comment:
I think we should use alias to give this variable a more meaningful name.

Reply:
I'm not sure I understand what you mean, this is typesafe and recommended by i18nnext

Reply:
oh okay, I didn't know that.